pkgs/tools/nix/nix-init/default.nix

```diff
@@ -12,7 +12,7 @@
 , callPackage
 , spdx-license-list-data
 }:
-
+  
```

Comment:
```suggestion

```

Reply:
The last 2 commits are there to test the action, I can revert them once this is approved